### PR TITLE
Tiny docs PR: simplified tree outputs in a few places

### PR DESCRIPTION
**Pull request by Aylr**

### CODE REVIEW

**Review by eugmandel — DISMISSED** [review]

File: docs/reference/implemented_expectations.rst [hunk 1]
```diff
@@ -45,11 +45,11 @@ out the missing implementations!
 +-----------------------------------------------------------------------------------+----------+----------+----------+
 |`expect_column_value_lengths_to_equal`                                             | True     | True     | True     |
 +-----------------------------------------------------------------------------------+----------+----------+----------+
-|`expect_column_values_to_match_regex`                                              | True     | False    | True     |
+|`expect_column_values_to_match_regex`                                              | True     | True     | True     |
 +-----------------------------------------------------------------------------------+----------+----------+----------+
 |`expect_column_values_to_not_match_regex`                                          | True     | False    | True     |
```
Comment (jcampbell) [inline docs/reference/implemented_expectations.rst]:
```suggestion
 |`expect_column_values_to_not_match_regex`                                          | True     | True     | True     |
```

File: docs/reference/implemented_expectations.rst [hunk 2]
```diff
@@ -45,11 +45,11 @@ out the missing implementations!
...
 +-----------------------------------------------------------------------------------+----------+----------+----------+
-|`expect_column_values_to_match_regex_list`                                         | True     | False    | False    |
+|`expect_column_values_to_match_regex_list`                                         | True     | True     | False    |
 +-----------------------------------------------------------------------------------+----------+----------+----------+
 |`expect_column_values_to_not_match_regex_list`                                     | True     | False    | False    |
```
Comment (jcampbell) [inline docs/reference/implemented_expectations.rst]:
```suggestion
 |`expect_column_values_to_not_match_regex_list`                                     | True     | True     | False    |
```

**Review by jcampbell — CHANGES_REQUESTED** [review]
Two small additional corrections to the implemented expectations table.

**Review by jcampbell — DISMISSED** [review]

File: docs/reference/implemented_expectations.rst [hunk 3]
```diff
@@ -45,13 +45,13 @@ out the missing implementations!
 +-----------------------------------------------------------------------------------+----------+----------+----------+
 |`expect_column_value_lengths_to_equal`                                             | True     | True     | True     |
 +-----------------------------------------------------------------------------------+----------+----------+----------+
-|`expect_column_values_to_match_regex`                                              | True     | False    | True     |
+|`expect_column_values_to_match_regex`                                              | True     | True     | True     |
 +-----------------------------------------------------------------------------------+----------+----------+----------+
-|`expect_column_values_to_not_match_regex`                                          | True     | False    | True     |
+ |`expect_column_values_to_not_match_regex`                                          | True     | True     | True     |
```
Comment (jcampbell) [inline docs/reference/implemented_expectations.rst]:
```suggestion
|`expect_column_values_to_not_match_regex`                                          | True     | True     | True     |
```

File: docs/reference/implemented_expectations.rst [hunk 4]
```diff
@@ -45,13 +45,13 @@ out the missing implementations!
...
 +-----------------------------------------------------------------------------------+----------+----------+----------+
-|`expect_column_values_to_match_regex_list`                                         | True     | False    | False    |
+|`expect_column_values_to_match_regex_list`                                         | True     | True     | False    |
 +-----------------------------------------------------------------------------------+----------+----------+----------+
-|`expect_column_values_to_not_match_regex_list`                                     | True     | False    | False    |
+ |`expect_column_values_to_not_match_regex_list`                                     | True     | True     | False    |
```
Comment (jcampbell) [inline docs/reference/implemented_expectations.rst]:
```suggestion
|`expect_column_values_to_not_match_regex_list`                                     | True     | True     | False    |
```

**Review by jcampbell — APPROVED** [review]

**Pull request merged**